app/routes.py

```diff
@@ -1,2 +1,131 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.route_helpers import validate_model
 
+bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                        description=request_body['description'],
+                        radius=request_body['radius']
+                        )
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
+
+@bp.route("", methods=["GET"])
+def read_all_planets():
+
+    name_param = request.args.get("name")
+    description_param = request.args.get("description")
+    radius_param = request.args.get("radius")
+
+    planet_query = Planet.query
+
+    if name_param:
+        planet_query = planet_query.filter_by(name=name_param)
+    if description_param:
+        planet_query = planet_query.filter(Planet.description.ilike(
+            f"%{description_param}%"))
+    if radius_param:
+        # filter planets with a radius greater than query param
+        # How can we give the option of filter lesser than or greater than?
+        # Would we need to prompt the user to include that param and then build
+        # conditionals to check if they want to find greater or less than?
+        planet_query = planet_query.filter(Planet.radius>radius_param)
+    
+    planets_response = [planet.to_dict() for planet in planet_query]
+    
+    return jsonify(planets_response)
+
+
+@bp.route("/<planet_id>", methods=["GET"])
+def read_planet(planet_id):
+    planet = validate_model(Planet, planet_id)
+    return planet.to_dict()
+
+@bp.route("/<planet_id>", methods=["PUT"])
+def update_planet(planet_id):
+    planet = validate_model(Planet, planet_id)
+
+    request_body = request.get_json()
+
+    planet.name = request_body["name"]
+    planet.description = request_body["description"]
+    planet.radius = request_body["radius"]
```

Comment:
As with my suggestion for the POST endpoint, we may want to wrap this with a `try`/`expect` to handle any `KeyError` that occurs.